code/modules/games/saintsandsins_cards.dm

```diff
@@ -182,7 +182,7 @@
 /decl/sas_card/heretic/exalted_atomus
 	name = "Exalted Atomus"
 	health = 17
-	ability = "ACTIVE: Declare a damage type at the start of caster's turn. Nullify that damage type for one round."
+	ability = "ACTIVE: Declare a damage type at the start of your turn. Nullify that damage type until the start of your next turn."
```

Comment:
This feels ambiguous about the scope of the spells being targeted but I can't express what I think it means succinctly. Does it nullify only the damage targeting the caster, or the damage targeting _any_ heretic?